fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/PushDownAggToMetaScanRule.java

```diff
@@ -126,7 +126,10 @@ public List<OptExpression> transform(OptExpression input, OptimizerContext conte
             }
 
             aggColumnIdToNames.put(metaColumn.getId(), metaColumnName);
-            newScanColumnRefs.put(metaColumn, metaScan.getColRefToColumnMetaMap().get(usedColumn));
+            Column c = metaScan.getColRefToColumnMetaMap().get(usedColumn);
+            // for empty table, meta scan may return NULL result, so we force set usedColumn to nullable.
+            c.setIsAllowNull(true);
+            newScanColumnRefs.put(metaColumn, c);
```

Comment:
soo, modify column-ref can't work?

Reply:
> modify column-ref can't work?

not work, seems nullable property depens on Column::isAllowNull, not column ref